primitives/state-machine/src/backend.rs

```diff
@@ -359,3 +359,22 @@ pub(crate) fn insert_into_memory_db<H, I>(mdb: &mut MemoryDB<H>, input: I) -> Op
 
 	Some(root)
 }
+
+/// Get the runtime code from the given `backend`.
+///
+/// Returns an error if the `:code` could not be found.
+pub fn get_runtime_code<H: Hasher, B: Backend<H>>(backend: &B) -> Result<RuntimeCode, &'static str>
+	where H::Out: Encode,
+{
+	let code = backend.storage(well_known_keys::CODE)
+			.ok()
+			.flatten()
+			.ok_or_else(|| "`:code` not found")?;
```

Comment:
nit: `.ok_or` can work here as well

```suggestion
			.ok_or("`:code` not found")?;
```